liiweb/templates/liiweb/legislation_list.html

```diff
@@ -29,6 +29,16 @@ <h1 class="my-4">{% trans 'Legislation' %}</h1>
               {% trans 'All legislation' %}
             </a>
           </li>
+          <li class="nav-item">
+            <a class="nav-link {% if view.variant == 'subleg' %}active{% endif %}" href="{% url 'legislation_list_regulations' %}">
+                {% if regulation_label %}
```

Comment:
2 spaces, not 4.

Can we configure black or one of our other precommit tools to enforce this?

Reply:
I don't think black works for html. Prettier could work but it doesn't work well with django templates. It breaks up the syntax causing errors. I will try and see if there is something else that can work. In any case this part of the code has changed.